source/common/http/http1/codec_impl.h

```diff
@@ -280,6 +295,10 @@ class ServerConnectionImpl : public ServerConnection, public ConnectionImpl {
   void onMessageComplete() override;
   void onResetStream(StreamResetReason reason) override;
   void sendProtocolError() override;
+  void onOutputBufferAboveHighWatermark() override;
+  void onOutputBufferBelowLowWatermark() override;
+
+  void resetActiveRequest();
```

Comment:
nit: move above line 289 (just so it's visually easy to see which are overrides).